osafw-app/App_Data/template/layout/sidebar.html

```diff
@@ -84,6 +84,11 @@ <h3>
         </a>
       </li>
       <~dev_menu if="GLOBAL[IS_DEV]" inline>
```

Comment:
IS_DEV is false on all non-local environments, so it won't be visible on live/beta servers.
I would do differently - put it out of IS_DEV block, for Site Admins only.
And only if there are any db updates pending (call parse and save to session in Users.doLogin)